### PR TITLE
Mobile structure setup

### PR DESCRIPTION
**Pull request by rbhatt1999**
In the first mobile structure setup branch following changes done-:
Ensured there was no linter error
Added a header, headline section & social media logo, and link in the index.html file.
Added the images in the icons folder.
Added styling for the header and headline section in the main.css file
Ensured to match template 2 from Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="./css/general.css">
+  <title>Rohit's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="title">
+      <h5>Rohit Bhatt</h5>
```
Comment (aimalamiri) [inline index.html:14]: Good job 👌but it's highly recommended to use the `a` tag for your logo, so please make sure to put your logo inside the `a` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    </div>
+    <nav class="menu">
+     <a href="#">
+      <img class="hamburger-menu" src="./icons/hamburger-menu.png" alt="">
+     </a> 
+    </nav>
+  </header>
+  <main class="main-container">
```
Comment (aimalamiri) [inline index.html:22]: You're almost done, but you can follow the Figma design for improvements. Here is what you can do.

- [ ] 1. Please make sure to follow the exact colors, fonts, and sizes. So in here, it's highly recommended to update your background color, and resize your background image to match the Figma template.
- [ ] 2. The content for the `h1` tag looks great, but you can improve its design by setting its `line-height` to `64px`.
- [ ] 3. The gap between social media icons is not similar to the one in Figma design, so kindly make sure to update its styles.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @rbhatt1999, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The header section is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (rbhatt1999) [reply index.html:22]: updated

**Review by codecaiine — APPROVED** [review]
Hi @rbhatt1999 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great UI design :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- Good use of github flow :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**